### DISCUSSION TITLE
Colored Markdown "notes" in Pluto

### DISCUSSION THREAD

**Discussion by jdc-pub** [fonsp/Pluto.jl#857] [Questions about using Pluto] [answered]
Is there any way to create colored Markdown "tips" or "notes", as shown in Julia's [Markdown Documentation](https://docs.julialang.org/en/v1/stdlib/Markdown/)? I think I've seen some Pluto notebooks on course channels on YouTube (mainly MIT courses), but I'm not sure how to do this. When I write `md"!!! note Note text goes here"`, no colored output is shown.  When I write...

```julia
md"""
!!! note
Note text goes here.
"""
```

I see a colored Markdown section with the header "note", but "Note text goes here" is printed below the section.

**Comment by icweaver** [comment, marked as answer]
Hey @cadojo! Yep, here are some of the popular ones from the [MIT course](https://computationalthinking.mit.edu/Fall20/) that you can play around with:

```julia
hint(text) = Markdown.MD(Markdown.Admonition("hint", "Hint", [text]))
```

```julia
keep_working(text=md"The answer is not quite right.") = Markdown.MD(Markdown.Admonition("danger", "Keep working on it!", [text]))
```

```julia
almost(text) = Markdown.MD(Markdown.Admonition("warning", "Almost there!", [text]))
```

```julia
correct(text=rand(yays)) = Markdown.MD(Markdown.Admonition("correct", "Got it!", [text]))
```

I think the one you posted just needs a tab to be displayed properly:
```julia
md"""
!!! note
    Note text goes here.
"""

```

**Reply by jdc-pub** [reply]
Awesome, thank you!
